### PR TITLE
Jupyter Executive Council

### PR DESCRIPTION
**Pull request by afshin**
**Background or context to help others understand the change.**

This PR contains the Executive Council document, which is the third major branch of the new Jupyter governance, following the adoption of the Distinguished Contributors and Software Steering Council pull requests. It also contains the bootstrapping process for the transition to the new Executive Council. And finally, it adds the Community Advisory Panel to the list of Standing Committees, which was missing in the original PR.

**A brief summary of the change.**

The draft is complete, but the PR still needs a document to outline how to bootstrap the Executive Council. Once that document is added to this branch, we can call a vote. But since this document is ready, a draft pull request allows for wider community input.

**The process ❗**

<details>
<summary>The process for changing the governance pages is as follows:</summary>
* Open a pull request **in draft state**. This triggers a discussion and iteration phase for your proposed changes.
* When you believe enough discussion has happened, **move the pull request to an active state**. This triggers a vote.
* During the voting phase, no substantive changes may be made to the pull request.
* The Steering Council will vote, and at the end of voting the pull request is merged or closed.

The discussion phase is meant to gather input and multiple perspectives from the community.

Make sure that the community has had an opportunity to weigh in on the change **before calling a vote**. A good rule of thumb is to ask several Steering Council members if they believe that it is time for a vote, and to let at least one person review the pull request for structural quality and typos.
</details>

## Votes

> All votes are limited in time to 4 weeks after the voting phase begins. At the end of 4 weeks, the proposal passes if 2/3 of the votes are in favor (fractions of a vote rounded up to the nearest integer); otherwise the proposal is rejected and the PR is closed. Prior to the four-week limit, if at least 80% of the Steering Council has voted and 2/3 of the votes are in favor (fractions of a vote rounded up to the nearest integer), the proposal passes.

**The voting period for the PR is from 1 July 2022 to 29 July 2022 anywhere on 🌍**

@afshin
- [x] YES
- [ ] NO

@blink1073
- [x] YES
- [ ] NO

@Carreau
- [x] YES
- [ ] NO

@damianavila
- [x] YES
- [ ] NO

@ellisonbg
- [x] YES
- [ ] NO

@fperez
- [x] YES
- [ ] NO

@ivanov
- [ ] YES
- [ ] NO

@jasongrout
- [x] YES
- [ ] NO

@jhamrick
- [ ] YES
- [ ] NO

@minrk
- [x] YES
- [ ] NO

@mpacer
- [ ] YES
- [ ] NO

@parente
- [x] YES
- [ ] NO

@rgbkrk
- [x] YES
- [ ] NO

@Ruv7
- [x] YES
- [ ] NO

@SylvainCorlay
- [x] YES
- [ ] NO

@takluyver
- [ ] YES
- [ ] NO

@willingc
- [x] YES
- [ ] NO

### CODE REVIEW

File: executive_council.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Nominate particular Working Groups to have a representative on the SSC.
+- Support and manage [Standing Committees](standing_committees_and_working_groups.md#standing-committees).
+- Actively work to increase and promote diversity, equity, and inclusion across the Project and its leadership bodies.
+- Maintain and implement the [Jupyter Code of Conduct](conduct/code_of_conduct.md) and incident response process.
+- Serve as the body of appeal for incident decisions made by the Code of Conduct Standing Committee.
+- Raise, allocate, and manage all Project funding and sponsorships, and set policies and procedures for third parties to request external funding with the official endorsement of Project Jupyter.
+- Allocate resources, select candidates, define scope and priorities, and manage roles and personnel employed to conduct business operations on behalf of Project Jupyter.
+- Manage legal aspects of the Project, such as copyright, contracts, compliance, and licensing.
+- Manage the Jupyter brand, including the Project-wide website, core design philosophy, design assets, trademark, and naming.
```
Comment (jasongrout) [inline executive_council.md:22]: We may have multiple trademarks:
```suggestion
- Manage the Jupyter brand, including the Project-wide website, core design philosophy, design assets, trademarks, and naming.
```

**Review by rgbkrk — APPROVED** [review]

**Comment by afshin** [comment]
cc: @jupyter/steeringcouncil

**Review by willingc — APPROVED** [review]

**Review by minrk — APPROVED** [review]

**Review by fperez — APPROVED** [review]
We've worked enough on this, so from my end it's a go.

**Review by ellisonbg — APPROVED** [review]

**Comment by fperez** [comment]
My approval here is meant only to convey the fact that I worked on this document extensively, so I'm happy seeing it go forward. I look forward to comments and discussion as we iterate on this, before we move to a formal vote.

**Review by SylvainCorlay — APPROVED** [review]

**Review by blink1073 — APPROVED** [review]

**Review by parente — APPROVED** [review]

**Review by damianavila — APPROVED** [review]

**Review by SylvainCorlay — APPROVED** [review]

**Review by willingc — APPROVED** [review]

**Review by Carreau — APPROVED** [review]

**Pull request merged**